python/adelphi/test-requirements.txt

```diff
@@ -1,5 +1,2 @@
-click ~= 7.1
-cassandra-driver ~= 3.24
 docker ~= 4.4
-tenacity ~= 7.0
```

Comment:
Now that test-adelphi is a full-fledged part of the Python package (see setup.py change above) there isn't any need to manage it's dependencies via test requirements. This change was one of the significant arguments for making test-adelphi part of the package.